src/main/java/net/wasdev/wlp/ant/install/WasDevInstaller.java

```diff
@@ -69,7 +69,7 @@ public void install(InstallLibertyTask task) throws Exception {
         }
 
         if (type == null) {
-            type = (licenseCode == null) ? "webProfile7" : "webProfile6";
+            type = "webProfile7";
```

Comment:
Is anything preventing the user from specifying webProfile6 for the type? I noticed another line of code in this class that references webProfile6 (method getRuntimeURI).

Reply:
And what if they were using an older version of Liberty? Wouldn't webProfile6 still be valid?

Reply:
The build should fail if the user specifies `webProfile6`, since it isn't under the list of recognized runtimes. The reason that the builds were defaulting to `wlp-runtime` is because the Ant task was approving the `webProfile6` runtime type, even though it doesn't exist anymore. 

I've cleaned up the logic in `getRuntimeURI` to correct this behavior.